extract_timeseries_tar.py

```diff
@@ -108,6 +226,21 @@ def create_timeseries_root_dir(file_entitiles, output_dir):
 
     return timeseries_root_dir
 
+def download_atlases():
+    """Download all atlases using ATLAS_METADATA."""
+
+    for atlas_name in ATLAS_METADATA.keys():
+        cur_atlas_meta = ATLAS_METADATA[atlas_name].copy()
+        if cur_atlas_meta['source'] == "templateflow":
+            for template in cur_atlas_meta['templates']:
+                for resolution in cur_atlas_meta['resolutions']:
+                    for dimension in cur_atlas_meta['dimensions']:
+                        description_keywords = {"dimension": dimension}
+                        print(f"-- {atlas_name}.{template}.{resolution}mm.{dimension}dim --")
+                        atlas = fetch_atlas_path(atlas_name,
+                                              resolution=resolution,
+                                              template=template,
+                                              description_keywords=description_keywords)
```

Comment:
Although all atlases have some kind of information related to dimension, `dimensions` is an user defined filed, not part of the templateflow convention. 
[See keyword arguments for templateflow file name entities.](https://www.templateflow.org/python-client/0.7.0/api/templateflow.api.html)
I would opt for a more universal solution. 
I see no harm just fetching the full templateflow archive. Or fetch all MNI152 templates:
```suggestion
    for template in templateflow.api.templates():
        if "MNI152" in template:
            templateflow.api.get(template)
```

Reply:
I like the suggestion, because indeed nested loop is not ideal...
But how do you handle cases where user uses another atlas ?
Maybe we remove the `if` statement then ?

Reply:
Yeah I thought about removing the if statement as well.
FYI, all atlases should be created for standard spaces, either MNI152 (volume metric) or freesurfer/fsLS (surface), unless they want to be really special and question their own meaning of life.
We have no surface data support so far.

Reply:
"unless they want to be really special and question their own meaning of life." I am asking the meaning of life everyday.
haha okay let's commit this, I will just test it before merge.